slsReceiverSoftware/src/DataProcessor.cpp

```diff
@@ -519,6 +501,45 @@ void DataProcessor::PadMissingPackets(char* buf) {
 	}
 }
 
+/** ctb specific */
+void DataProcessor::RearrangeDbitData(char* buf) {
```

Comment:
Overall in this function I think we could get away with less pointers and more RAII and pass by value. But we can leave this for refactoring

Reply:
The pointer style was to do less work from the client interface (all the parameters are passed this way), but I had been thinking of changing that to make it more clean. This will have to be done at a different time and yes, it would be refactoring.